libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -572,6 +574,25 @@ void RC_Channel::do_aux_function_camera_trigger(const aux_switch_pos_t ch_flag)
     }
 }
 
+void RC_Channel::do_aux_function_runcam_control(const aux_switch_pos_t ch_flag)
+{
+    AP_RunCam *runcam = AP::runcam();
+    if (runcam == nullptr) {
+        return;
+    }
+
+    switch (ch_flag) {
+        case HIGH:
+            // high switch starts recording
+            runcam->start_recording();
+            break;
+        default:
+            // low switch stops recording
```

Comment:
Pointless (and insufficient as it doesn't mention it also handles `case MIDDLE`) comment.

Reply:
Ok